int64 id
int64 com_id
int64 com_baud
---
bool success
int64[] response